dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -233,9 +235,12 @@ def generate_workshops_survey_summary(workshop)
               end
               session_summary[:facilitator][q_key] = facilitator_response_averages
             else
-              # For non facilitator specific responses, just return the average
-              sum = surveys_for_session.map {|survey| survey[response_section][q_key]}.reduce(0, :+)
-              session_summary[response_section][q_key] = (sum / surveys_for_session.size.to_f).round(2)
+              # For non facilitator specific responses, just return a frequency map with
+              # nulls removed
+              # [1, 1, 2, 2, 3, 5, 7, 7, 7, 7, 7, nil, nil] => {1: 2, 2: 2, 3: 1, 5: 1, 7: 5}
+              summary = Hash[*surveys_for_session[response_section].map {|survey| survey[q_key]}.group_by {|v| v}.flat_map {|k, v| [k, v.size]}]
```

Comment:
As discussed offline, please remove the `Hash[*` and switch `flat_map` to `transform_values`. That should make it more readable. Thanks!

Reply:
Or perhaps even clearer, something like this:
```ruby
summary  = surveys_for_session[response_section].each_with_object(Hash.new(0)) {|v, counts| counts[v] += 1}
```